include/envoy/stats/histogram.h

```diff
@@ -17,16 +17,6 @@ class HistogramStatistics {
 public:
   virtual ~HistogramStatistics() {}
 
-  /**
```

Comment:
These were only used in tests, to equality-compare `HistogramStatistics` objects.